docs/docs.trychroma.com/pages/integrations/jinaai.md

```diff
@@ -14,8 +14,6 @@ Chroma provides a convenient wrapper around JinaAI's embedding API. This embeddi
 {% tabs group="code-lang" hideTabs=true %}
 {% tab label="Python" %}
 
-This embedding function relies on the `requests` python package, which you can install with `pip install requests`.
```

Comment:
I don't think these notes are relevant since chromadb requires httpx for both the fat/thin clients

Reply:
I think these are internal to the EF?

Reply:
right, but these docs are assuming chromadb and all sub dependencies were installed (I think) in which case httpx is already available and doesn't need to be explicitly installed separately

Reply:
I guess I am confused - this is about requests not httpx?

Reply:
on main, all embedding functions use `requests`
this PR updates all embedding functions to use `httpx`